ui/e2e/src/tests/panelEditor.spec.ts

```diff
@@ -51,4 +51,24 @@ test.describe('Dashboard: Panel Editor', () => {
     await expect(discardChangesConfirmationDialog).toBeHidden();
     await panelEditor.isClosed();
   });
+
+  test('should reset y axis panel option to default', async ({ dashboardPage }) => {
+    await dashboardPage.startEditing();
+    await dashboardPage.addPanel();
+    const panelEditor = dashboardPage.getPanelEditor();
+    await panelEditor.isVisible();
+    const settingsTab = dashboardPage.page.getByRole('tab', { name: 'Settings', exact: true });
+    await settingsTab.click();
+    const resetButton = dashboardPage.page.getByRole('button', { name: 'Reset To Defaults', exact: true });
```

Comment:
I think it makes sense to leave it in here for now if it's not a common feature in the panel editor. Once it's in all/most of the panel plugins, it could be nice to add as a reusable helper in there.